flake.nix

```diff
@@ -23,12 +30,17 @@
       devShell.x86_64-linux = import ./devShell.nix { system = "x86_64-linux"; };
 
       lib = flake-utils.lib // {
-        # modulesFromList is deprecated, will be removed in future releases
-        inherit systemFlake modulesFromList;
+        inherit (internal-functions) rakeLeaves;
```

Comment:
I don't think it's supposed to be here?

Reply:
Why not? Where should it be then?

I thought we were removing deprecations in a later PR.

Reply:
I mean the `rakeLeaves` function. I do not see a reason why it should not be internal only since modules exporter takes care of use-cases involving this function

Reply:
@blaggacao said they use it a lot at DevOS and digga. So it's better to have it top-level.

Reply:
I got a feeling that they themselves would prefer using something along the lines of the proxy pattern. I'd say there is no reason to export it externally, thus lets avoid that. Especially since we already have an `internal` namespace

Reply:
@blaggacao thoughts?

Reply:
I'm fine with proxying via the `internal` namespace. These projects are so close, that we probably would not have issue with drift.

Reply:
Okay, fixing now.